src/auto-evo/simulation/PopulationSimulation.cs

```diff
@@ -207,7 +221,7 @@ private static List<Species> CopyInitialPopulationsToResults(SimulationConfigura
                     / energyBalanceInfo.FinalBalanceStationary);
 
                 // Severely penalize a species that can't osmoregulate
-                if (energyBalanceInfo.FinalBalance < 0)
+                if (energyBalanceInfo.FinalBalanceStationary < 0)
```

Comment:
It's badly labeled, but my reasoning for this was that most AI is always moving, so the moving ATP balance is the one that counts. IIRC I noticed a lot of AI microbes that just immediately killed themselves in motion in 0.5.5, so if you want to keep this change it would be wise to test for a while to make sure you don't get self-starving microbes.

Personally, I would suggest just not making this change at all. This doesn't address the "emergency" since players weren't complaining about ATP-negative species not doing well in auto-evo, and an AI that kills itself is one of those "stupid" things that really stand out to players.

A middle ground would be to do a sessile check, looking at the Activity attribute and picking a balance based on being in the bottom tenth range. That would have the drawback of being brittle in the future though.

Reply:
> Personally, I would suggest just not making this change at all. This doesn't address the "emergency" since players weren't complaining about ATP-negative species not doing well in auto-evo, and an AI that kills itself is one of those "stupid" things that really stand out to players.

I disagree in that it often seems that the auto-evo *really* wants the player to go extinct. Which would make sense if they triggered this divide by 10 line.
Shouldn't the AI by this point know how to stop to recharge ATP?

Reply:
Sadly it does not, since the AI refactors never got around to "out of ATP vs starving and need to find food" and "want to run away vs need to run away" calculations.

A compromise option might be to hold the player and AI to different rules, letting the more responsible humans play with ATP-flickering species but not the AI. Even then though the player's sibling will likely run off and die.

Reply:
I guess technically we could check the PlayerSpecies flag here... well it's not too dirty for me to consider putting it in this PR.

Reply:
We're in a tough position: the same mechanic needs to do different things for the player and AI. I'm sorry I didn't think about this when you first made a PR bringing auto-evo into the player experience.

Reply:
I added in the hack I mentioned.